src/core/lib/iomgr/tcp_server_posix.cc

```diff
@@ -530,6 +709,12 @@ static void tcp_server_start(grpc_tcp_server* s,
   GPR_ASSERT(s->on_accept_cb);
   GPR_ASSERT(s->active_ports == 0);
   s->pollsets = pollsets;
+  if (grpc_event_engine::experimental::UseEventEngineServer()) {
+    GPR_ASSERT(!s->shutdown_listeners);
+    (void)s->ee_listener->Start();
```

Comment:
We should not bury this error, it may be meaningful when other posix-compatible EventEngines are used.

Reply:
Logged this error now.